src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -223,6 +225,14 @@ public function setMaxDepthHandler(?callable $handler): void
         $this->maxDepthHandler = $handler;
     }
 
+    /**
+     * Allows to set a global context
+     */
+    public function setContext($context = array())
```

Comment:
i think after #28709 we have a better base to do this..

Reply:
nice, I hesitated between this and modifying constructor args but I think that this is still better.

Reply:
im not sure, we should it be mutable. here we obtain the defaults once from config (to me that's the missing piece in #28709 actually)

also we should consider defaults, per format

Reply:
true, I'll see if I can add this on #28709 directly

> also we should consider defaults, per format

Looks harder to implement and I'd even consider this as another feature.

Reply:
im not sure, i think it's part of the feature proposed here

Reply:
Yes it'll be but in another patch where we will allow a factory to instantiate multiple serializers if I understand things correctly.